src/display/canvas.js

```diff
@@ -566,7 +572,13 @@ function compileType3Glyph(imgData) {
     if (p === end) {
       continue;
     }
-    const coords = [p % width1, i];
+
+    if (path) {
+      path.moveTo(p % width1, i);
+    } else {
+      // eslint-disable-next-line no-var
+      var coords = [p % width1, i];
```

Comment:
Why a `var` ? Why not adding `coords`  to `let path, outlines` ?

Reply:
Mostly to avoid, by default, defining a useless variable.
However, I suppose that it doesn't matter in the grand scheme of things so it's changed now :-)